src/api.v2/controllers/experimentController.js

```diff
@@ -129,11 +138,45 @@ const downloadData = async (req, res) => {
   logger.log(`Providing download link for download ${downloadType} for experiment ${experimentId}`);
 
   const downloadLink = await new Experiment().getDownloadLink(experimentId, downloadType);
+
+  logger.log(`Finished providing download link for download ${downloadType} for experiment ${experimentId}`);
   res.json(downloadLink);
 };
 
+
+const cloneExperiment = async (req, res) => {
+  const getAllSampleIds = async (experimentId) => {
+    const { samplesOrder } = await new Experiment().findById(experimentId).first();
+    return samplesOrder;
+  };
+
+  const {
+    params: { experimentId: fromExperimentId, toExperimentId },
```

Comment:
is the new experiment then created in the UI?

Reply:
Based on what the ticket said I understood we created an empty experiment, the experiment showed us the optional datasets and then the samples get cloned into this empty experiment. So the experiment is already created because that's where we get the options of example datasets
<img width="1364" alt="image" src="https://user-images.githubusercontent.com/8465903/175292477-19f004f5-580b-4e04-b4db-128702173f1a.png">

Reply:
Discussed and alreayd adapted the code to what we resolved, create a new experiment and leave this experiment empty